packages/migrate/src/__tests__/__helpers__/prismaConfig.ts

```diff
@@ -53,6 +53,9 @@ function defaultTestConfig(ctx: BaseContext): PrismaConfigInternal {
   })
 }
 
+/**
+ * TODO: rewrite this to load this via `@prisma/config`'s `loadConfigFromFile` function. 
+ */
```

Comment:
https://linear.app/prisma-company/issue/ORM-1246/prismaconfig-let-fixture-tests-in-migrate-and-cli-packages-use